src/explainer.ts

```diff
@@ -3,18 +3,31 @@
 import request = require('request');
 import * as kubernetes from '@kubernetes/client-node';
 import * as pluralize from 'pluralize';
+import * as shelljs from 'shelljs';
+
 import { formatComplex, formatOne, Typed, formatType } from "./schema-formatting";
-import { getActiveKubeconfig } from './components/config/config';
+import { getActiveKubeconfig, getUseWsl } from './components/config/config';
 
-export function readSwagger(): Promise<any> {
+function loadKubeconfig(): kubernetes.KubeConfig {
     const kc = new kubernetes.KubeConfig();
     const kubeconfig = getActiveKubeconfig();
     if (kubeconfig) {
         kc.loadFromFile(kubeconfig);
     } else {
+        if (getUseWsl()) {
+            const result = shelljs.exec('wsl.exe cat $HOME/.kube/config', { silent: true }) as shelljs.ExecOutputReturnValue;
```

Comment:
Do we need to handle the case where they have WSL, have `kubectl` under WSL, but use a KUBECONFIG under WSL to use a non-default kubeconfig file?

Reply:
done.